sites/cheerpx/src/content/docs/13-tutorials/setup-svelte.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Svelte setup
+description: Setting up Svelte app
+---
+
+This tutorial will explain how to create a application with Svelte that works together with CheerpX.
+
+## Install Svelte
+
+First create a new Svelte app by running:
+
+```
+npm create svelte@latest my-app
+```
+
+1. Select the Skeleton Project option.
+2. Enable Typescript if needed.
+3. There are no additional options required for this tutorial.
+
+Next, navigate to your project directory and install the dependencies:
+
+```
+cd my-app
+npm install
+npm run dev
+```
+
+At this point, the development server will run from another terminal in the background.
+
+We're able to test the development server by visiting: http://localhost:5173/
+
+## Install Cheerpx
+
+Next, let's install CheerpX.
+
+This [link] will direct you to the installation page of CheerpX.
```

Comment:
CheerpX is "installed" (so to speak, the NPM module just loads a public CDN build) directly via npm

`npm install @leaningtech/cheerpx`

Reply:
Also, please be careful, the link goes to the _Cheerp_ install page. CheerpX is not Cheerp

Reply:
I removed the link and added the npm install